### PR TITLE
Changing default colors to match 18F colors

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Based on https://github.com/cfpb/capital-framework/issues/193, I've gone ahead and pulled in 18F's colors from the US Federal Web Design Standards to replace most of the defaults in this repo. We'll continue to use the CFPB colors for our projects, but this will make our documentation look nicer in the meantime and provide nicer defaults for new projects.

### CODE REVIEW

**Comment by ascott1** [comment]
Yay! This is awesome. +1 form me. @Scotchester do you want to take a look as well?

File: src/cf-vars.less [hunk 1]
```diff
@@ -72,33 +72,33 @@
 // Color variables
```
Comment (Scotchester) [inline src/cf-vars.less:72]: Maybe add a quick note up here explaining what the various `$color-` comments below are referring to?

File: src/cf-vars.less [hunk 2]
```diff
@@ -72,33 +72,33 @@
...
-@link-text-active:              darken(@link-text, 15%);
-@link-underline-active:         darken(@link-underline, 15%);
+@link-text:                     #0071bc; // $color-primary
+@link-underline:                #0071bc; // $color-primary
+@link-text-visited:             #4c2c92; // $color-visited
+@link-underline-visited:        #4c2c92; // $color-visited
+@link-text-hover:               #205493; // $color-primary-darker
+@link-underline-hover:          #205493; // $color-primary-darker
+@link-text-active:              #205493; // $color-primary-darker
+@link-underline-active:         #205493; // $color-primary-darker
```
Comment (Scotchester) [inline src/cf-vars.less]: I feel like we should break from what they've implemented currently to have a different active state. I suggest `$color-primary-darkest`.

File: src/cf-vars.less [hunk 3]
```diff
@@ -72,33 +72,33 @@
...
-@thead-text:                    #000;
-@thead-bg:                      #f8f8f8;
+@thead-text:                    @text;
+@thead-bg:                      #f1f1f1; // $color-gray-lightest:
 
 // input
-@input-bg:                      #fff;
-@input-border:                  darken(#895983, 15%);
-@input-border-focus:            #c7336e;
+@input-bg:                      #ffffff;
```
Comment (Scotchester) [inline src/cf-vars.less]: Our [style guide](https://github.com/cfpb/front-end/blob/master/css.md#property-values) implies a preference for use three-character hex values when possible.

File: src/cf-vars.less [hunk 4]
```diff
@@ -72,33 +72,33 @@
...
+@input-border:                  #5b616b; // $color-gray:
+@input-border-focus:            #3E94CF; // $color-focus:
```
Comment (Scotchester) [inline src/cf-vars.less]: Uncapitalize the letters in the hex value here for consistency.

File: src/cf-vars.less [hunk 5]
```diff
@@ -72,33 +72,33 @@
...
 @input-placeholder:             grayscale(#c7336e);
 
 // .figure__bordered
-@figure__bordered:              darken(#895983, 15%);
+@figure__bordered:              #d6d7d9; // $color-gray-lighter:
```
Comment (Scotchester) [inline src/cf-vars.less]: Trivial: The last four comments have colons at the end of them.

**Comment by Scotchester** [comment]
A few minor things, but glad to see this PR. Thanks for getting the ball rolling on this, @KimberlyMunoz!

File: src/cf-vars.less (hunk 1 above)
Comment (jimmynotjim) [inline src/cf-vars.less:72]: Do we necessarily need them? Are we going to update these if 18F changes any of them in the future? Or is this just a starting point and we'll go on our own from here?

File: src/cf-vars.less (hunk 2 above)
Comment (KimberlyMunoz) [inline src/cf-vars.less]: To confirm this is for `@link-text-active` and `@link-underline-active`, right?

**Comment by KimberlyMunoz** [comment]
Most/all of the comments should be addressed in 15a1b8b. Thanks @Scotchester!

File: src/cf-vars.less (hunk 2 above)
Comment (Scotchester) [inline src/cf-vars.less]: Yes, that's right.

**Comment by Scotchester** [comment]
Looking great; thanks! :+1:

**Pull request merged**